sys/aarch64/pmap.c

```diff
@@ -82,6 +82,7 @@ static const pte_t vm_prot_map[] = {
 
 static pmap_t kernel_pmap;
 paddr_t _kernel_pmap_pde;
+static vaddr_t pmap_maxkvaddr;
```

Comment:
There is no difference between them now. That implementation is based on NetBSD version where `pmap_maxkvaddr` is private for pmap and uvm manages their own kernel end using return value of `pmap_growkernel`. On the other hand FreeBSD shares the same variable with other subsystems globally.

So now I'm not sure what is your expectation.

Reply:
I wasn't aware that NetBSD had two different variables for `mavkvaddr`. I assumed that they have different meanings, otherwise there would be no need to have two different variables. You could add a description here like "`pmap_maxkvaddr` is used to track the range of virtual kernel addresses for which page table pages have been allocated".